install.sh

```diff
@@ -0,0 +1,58 @@
+#/
+# Copyright 2015-2016 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#/
+#!/bin/bash
+#
+# use the command line interface to install pushnotifications package.
+# this script is in blue because it need blue whisk.properties.
+#
+
+#: ${WHISK_SYSTEM_AUTH:?"WHISK_SYSTEM_AUTH must be set and non-empty"}
+AUTH_KEY=`cat ../../../credentials/auth.whisk.system`
+
+SCRIPTDIR="$(dirname "$0")"
+CATALOG_HOME='../'
+source "$CATALOG_HOME/util.sh"
```

Comment:
make `install.sh` work stand-alone without requiring `util.sh` or the need for `$CATALOG_HOME`